docs/docs-source/docs/modules/get-started-gcp-marketplace/pages/index.adoc

```diff
@@ -0,0 +1,17 @@
+= Get started with Cloudflow on Google Cloud Platform Marketplace
+:toc:
+:toc-title: ON THIS PAGE
+:toclevels: 2
+
+include::ROOT:partial$include.adoc[]
+
+A convenient way to try out Cloudflow is via Google Cloud Platform (GCP) Marketplace, a place where you can quickly deploy Cloudflow that runs on Google Kubernetes Engine (GKE) with just a few clicks.
```

Comment:
Maybe "try out" sounds too temporary? It's intended to be a way to deploy and run full applications for production?

Suggestion: "A convenient way to try out and use Cloudflow...", or ""A convenient way to use Cloudflow..."

Reply:
I thought our Marketplace offering wasn't intended for production use cases. But the many thumbs up proved me wrong ;) I'll change the wording.